### DISCUSSION TITLE
How to request permission to contribute to the Sigma rules for azure

### DISCUSSION THREAD

**Discussion by Corissalea** [SigmaHQ/sigma#3045] [Q&A] [answered]
@thomaspatzke  - I am working on a guide to translate Azure AD SecOps rules to Sigma. I was unable to submit a Pull Request to the sigma repository, so I am wondering if there is a process for requesting permissions.
![git error](https://user-images.githubusercontent.com/33907780/170147268-bc64b4f6-e7e7-4ac0-93e4-733fd74d0c32.png)

**Comment by frack113** [comment, marked as answer]
Hello,
You have to : 
- make a [forks](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks)
- make a branch  with the change
- make a PR from your forks to SigmaHQ

To make a fork click on `Fork`
![image](https://user-images.githubusercontent.com/62423083/170182540-cc383206-fc1a-44a5-81d2-74001767cea8.png)
